src/BenchmarksServer/Startup.cs

```diff
@@ -129,6 +129,8 @@ public static int Main(string[] args)
                             Debug.Assert(tempDir == null);
                             tempDir = GetTempDir();
 
+                            var targetFramework = job.Framework == Framework.Core ? "netcoreapp1.1" : "net451";
```

Comment:
Doesn't this dupe `GetTFM()`?

Suggest `net452` in `GetTFM()` and here (if the code must be duplicated). Same for the Benchmarks.csproj file.

Hmm, why was the Benchmarks project already compiled for desktop framework? 😺